File: 1-Iniciante/Ex1009.java
import java.util.Scanner;

public class Ex1009 { // para ser aceito no URI mude o nome da classe para Main
    public static void main(String args[]) {
        /* variaveis */
        Scanner scanner = new Scanner(System.in);
        String nome = new String();
        Double salarioFixo, totalVendas, salarioFinal;

        /* entrada */
        nome = scanner.nextLine();
        salarioFixo = scanner.nextDouble();
        totalVendas = scanner.nextDouble();

        /* processamento */
        salarioFinal = salarioFixo + totalVendas * 0.15;

        /* saida */
        System.out.printf("TOTAL = R$ %.2f\n", salarioFinal);

        scanner.close();
    }
}

File: 1-Iniciante/Ex1009.kt
import java.util.Scanner

class Vendedor(
    val nome: String,
    val totalVendas: Double,
    val salarioFixo: Double
) {

    var salarioComBonus: Double = this.salarioFixo + (this.totalVendas * 0.15)
        private set
}

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    val vendedor = Vendedor(
        nome = scanner.nextLine(),
        salarioFixo = scanner.nextDouble(),
        totalVendas = scanner.nextDouble()
    )

    println("TOTAL = R$ ${String.format("%.2f", vendedor.salarioComBonus)}")
}
